src/view/abstract-element.ts

```diff
@@ -0,0 +1,25 @@
+import { createElement } from '../render';
+
+export interface TemplateElement {
+  template: string;
+  element: Element;
+  removeElement: () => void;
+}
+
+export abstract class AbstractElement {
+  _element: Element | null = null;
```

Comment:
К сожалению у вас есть критерий, что надо юзать ванильные приватные поля. Ну и как раз везде дженерик юзай верхний
```suggestion
  #element: E | null = null;
```